proof/refine/RISCV64/ArchAcc_R.thy

```diff
@@ -334,8 +334,9 @@ lemma corres_cross_over_pte_at:
   done
 
 lemma getObject_PTE_corres[corres]:
-  "corres pte_relation' (pte_at p and pspace_aligned and pspace_distinct) \<top>
-          (get_pte p) (getObject p)"
+  "p = p' \<Longrightarrow>
+   corres pte_relation' (pte_at p and pspace_aligned and pspace_distinct) \<top>
+          (get_pte p) (getObject p')"
```

Comment:
why not use `[@lift_corres_args]` on original lemma?

Reply:
I'm mostly trying to establish this form as the canonical form for writing corres lemmas to reduce friction. We could also state other lemmas with object implication and then use them with `rule_format`, but we don't do that, because it would be annoying to use `[rule_fromat]` everywhere.